Will review what's wrong, but now tests don't pass:

```
FAILED tests/components/fritz/test_config_flow.py::test_user[True-user_input0-expected_config0] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_user[False-user_input1-expected_config1] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_user[False-user_input2-expected_config2] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_user_already_configured[True-user_input0] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_user_already_configured[False-user_input1] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_security[True-user_input0-FritzAuthorizationError] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_security[True-user_input0-FritzSecurityError] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_security[False-user_input1-FritzAuthorizationError] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_security[False-user_input1-FritzSecurityError] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_connection[True-user_input0] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_connection[False-user_input1] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_unknown[True-user_input0] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_exception_unknown[False-user_input1] - Failed: Translation not found for fritz: `config.step.user.data_description.username`. Please add to homeassist...
FAILED tests/components/fritz/test_config_flow.py::test_reauth_successful - Failed: Translation not found for fritz: `config.step.reauth_confirm.data_description.username`. Please add to ...
FAILED tests/components/fritz/test_config_flow.py::test_reauth_not_successful[FritzAuthorizationError-invalid_auth] - Failed: Translation not found for fritz: `config.step.reauth_confirm.data_description.username`. Please add to ...
FAILED tests/components/fritz/test_config_flow.py::test_reauth_not_successful[FritzConnectionException-cannot_connect] - Failed: Translation not found for fritz: `config.step.reauth_confirm.data_description.username`. Please add to ...
FAILED tests/components/fritz/test_config_flow.py::test_reauth_not_successful[FritzSecurityError-invalid_auth] - Failed: Translation not found for fritz: `config.step.reauth_confirm.data_description.username`. Please add to ...
FAILED tests/components/fritz/test_config_flow.py::test_reconfigure_successful[True-user_input0-expected_config0] - Failed: Translation not found for fritz: `config.step.reconfigure.data_description.ssl`. Please add to homeassi...
FAILED tests/components/fritz/test_config_flow.py::test_reconfigure_successful[True-user_input1-expected_config1] - Failed: Translation not found for fritz: `config.step.reconfigure.data_description.ssl`. Please add to homeassi...
FAILED tests/components/fritz/test_config_flow.py::test_reconfigure_successful[True-user_input2-expected_config2] - Failed: Translation not found for fritz: `config.step.reconfigure.data_description.ssl`. Please add to homeassi...
FAILED tests/components/fritz/test_config_flow.py::test_reconfigure_successful[False-user_input3-expected_config3] - Failed: Translation not found for fritz: `config.step.reconfigure.data_description.ssl`. Please add to homeassi...
FAILED tests/components/fritz/test_config_flow.py::test_reconfigure_successful[False-user_input4-expected_config4] - Failed: Translation not found for fritz: `config.step.reconfigure.data_description.ssl`. Please add to homeassi...
FAILED tests/components/fritz/test_config_flow.py::test_reconfigure_not_successful - Failed: Translation not found for fritz: `config.step.reconfigure.data_description.ssl`. Please add to homeassi...
FAILED tests/components/fritz/test_config_flow.py::test_ssdp_already_in_progress_host - Failed: Translation not found for fritz: `config.step.confirm.data_description.username`. Please add to homeass...
FAILED tests/components/fritz/test_config_flow.py::test_ssdp - Failed: Translation not found for fritz: `config.step.confirm.data_description.username`. Please add to homeass...
FAILED tests/components/fritz/test_config_flow.py::test_ssdp_exception - Failed: Translation not found for fritz: `config.step.confirm.data_description.username`. Please add to homeass...
FAILED tests/components/fritz/test_config_flow.py::test_options_flow - Failed: Translation not found for fritz: `options.step.init.data_description.consider_home`. Please add to home...
FAILED tests/components/fritz/test_init.py::test_options_reload - Failed: Translation not found for fritz: `options.step.init.data_description.consider_home`. Please add to home...
FAILED tests/components/fritz/test_init.py::test_setup_auth_fail[FritzAuthorizationError] - Failed: Translation not found for fritz: `config.step.reauth_confirm.data_description.username`. Please add to ...
FAILED tests/components/fritz/test_init.py::test_setup_auth_fail[FritzSecurityError] - Failed: Translation not found for fritz: `config.step.reauth_confirm.data_description.username`. Please add to ...

Results (25.72s):
      70 passed
      30 error
```

--- a/homeassistant/components/fritz/strings.json
+++ b/homeassistant/components/fritz/strings.json
@@ -1,7 +1,7 @@
 {
   "common": {
     "data_description_host": "The hostname or IP address of your FRITZ!Box router.",
-    "data_description_port": "Leave it empty to use the default port.",
+    "data_description_port": "Leave empty to use the default port.",
     "data_description_username": "Username for the FRITZ!Box.",
     "data_description_password": "Password for the FRITZ!Box.",
     "data_description_ssl": "Use SSL to connect to the FRITZ!Box."

